Dockerfile_dev

```diff
@@ -10,14 +10,8 @@ COPY requirements.txt /tmp/
 # * Install system requirements
 # * Install pip requirements
 # * Empty system cache to conserve some space
-RUN zypper -n in python310-devel python310-pip gcc libffi-devel && pip3.10 install -r /tmp/requirements.txt && rm -rf /var/cache
+RUN zypper -n in python310-devel gcc libffi-devel && pip3.10 install -r /tmp/requirements.txt && rm -rf /var/cache
 
 WORKDIR /pcw
 
-## Finalize ################################################################# ##
-
-VOLUME /pcw/db
-
-EXPOSE 8000/tcp
-
-ENTRYPOINT ["sh", "-c", "/pcw/container-startup", "$@"]
+ENTRYPOINT ["sh", "-c", "/pcw/container-startup updaterun"]
```

Comment:
```suggestion
ENTRYPOINT ["sh", "-c", "/pcw/container-startup"]
```
I don't see a reason for this argument in `ENTRYPOINT`. I think that it makes sense to use this container file like:
```bash
podman build -t pcw_dev .
podman run -it --rm pcw_dev updaterun
podman run -it --rm pcw_dev removeclusters
podman run -it --rm pcw_dev orderquatroformagipizza
```

Reply:
the problem that it does not work like that : 
```
podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel updaterun                       [15:11:27]
pcw startup helper
Usage: /pcw/container-startup COMMAND
COMMANDS:
   help                    Print help message
   check                   Run system health checks
   run                     Standart publiccloud-watcher run procedure
   migrate                 Run database migration
   createsuperuser         Create superuser. Requires --email and --username argument
	 updaterun 							 Do a regular instance scan and persist results to local DB
	 rmclusters							 Scan whole AWS account and delete ANY EKS clusters

All other commands will be passed as arguments to 'manage.py'
```
 ( when I apply your suggestion and re-build the container )

Reply:
Pavel suggestion works, you only need to provide the additional commands as argument in your `podman run` command just as he suggested:

    podman build -t pcw_dev .
    podman run -it --rm pcw_dev updaterun
    podman run -it --rm pcw_dev removeclusters
    podman run -it --rm pcw_dev orderquatroformagipizza

Also :+1: for the :pizza: @pdostal :wink: Me gusta!

Reply:
why is not working for me ? 
```
podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel updaterun                       [15:11:27]
pcw startup helper
Usage: /pcw/container-startup COMMAND
COMMANDS:
   help                    Print help message
   check                   Run system health checks
   run                     Standart publiccloud-watcher run procedure
   migrate                 Run database migration
   createsuperuser         Create superuser. Requires --email and --username argument
	 updaterun 							 Do a regular instance scan and persist results to local DB
	 rmclusters							 Scan whole AWS account and delete ANY EKS clusters

All other commands will be passed as arguments to 'manage.py'
```

again I am running "podman run  -v /etc/pcw.ini:/etc/pcw.ini -v /var/pcw/:/var/pcw -v /home/asmorodskyi/source/pcw/:/pcw/  -t pcw-devel *updaterun*"